src/binary-search-tree.js

```diff
@@ -1,7 +1,20 @@
-// https://msdn.microsoft.com/en-us/library/aa289150(v=vs.71).aspx
-/* eslint-disable global-require */
-/* eslint-disable no-unused-vars */
-/* eslint-disable no-trailing-spaces */
+class Queue {
```

Comment:
You should be importing this class from its own file instead of pasting the code here in this file.